### ISSUE TITLE
Detect the missing data in the manifest during the validation

### ISSUE THREAD

**Issue by rrchai** [Sage-Bionetworks/schematic#587]
**Is your feature request related to a problem? Please describe.**
When submitting a manifest that has data deleted will pass the validation, but crash the app during submission
```
Warning: Error in py_call_impl: SynapseHTTPError: 404 Client Error: 
Entity syn25892968 is in trash can.
```

**Describe the solution you'd like**
I suggest it should be detected in the validation

**How important is this feature?** Select from the options below:
• 🌗 Medium - can do work without it; but it's important (e.g. to save time or for convenience)

**When will use cases depending on this become relevant?** Select from the options below:
• Mid-term - 2-4 months

**Additional context**
To reproduce:
1. Go to data curator app
2. Select "HATN Duke" > "imaging_level_2" > "Imaging Level 2"
3. validate and submit [imaging_level_2.csv](https://github.com/Sage-Bionetworks/schematic/files/8144197/imaging_level_2.csv)

**Comment by milen-sage** [comment]
@adamjtaylor we are reprioritizing this issue from sprint 15 to the current sprint 14. This would address problems related to synapse entity IDs that no longer exist.

**Comment by ychae** [comment]
Additionally: skip the deleted entity and provide a warning that that file no longer exists

**Comment by ychae** [comment]
Related: https://github.com/Sage-Bionetworks/data_curator/issues/188

**Comment by GiaJordan** [comment]
Following up on closure, annotation is skipped for deleted entities during the annotation process
@adamjtaylor @rrchai
